src/map/script.cpp

```diff
@@ -26528,6 +26528,41 @@ BUILDIN_FUNC(item_enchant){
 #endif
 }
 
+
+BUILDIN_FUNC(addfame) {
+	struct map_session_data *sd;
+
+	if (!script_charid2sd(3, sd))
+		return SCRIPT_CMD_FAILURE;
+
+	pc_addfame(*sd, script_getnum(st, 2));
```

Comment:
```suggestion
	if( !pc_addfame(*sd, script_getnum(st, 2)) ){
		return SCRIPT_CMD_FAILURE;
	}
```
This means that the command was called on a player that did not have any fame relevant job.
So in my eyes a bad script that did not check it and should be reported.